release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-chrony-chrony-default"]
```

Comment:
Tiny thing: do you need the double `chrony` in the id?

Reply:
Whoa, nope! Thanks!